src/driverApp/editValidation.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Try to find a better solution instead of disabling `eslint`. You can change add the corresponding rule in `eslint.config`.